tests/test_kratos_io.py

```diff
@@ -486,4 +508,99 @@ def test_write_mdpa_two_conditions_same_position(self):
         # check if mdpa data is as expected
         npt.assert_equal(actual=actual_mdpa_text, desired=expected_mdpa_text)
 
+    def test_write_project_parameters_with_spring_damper_and_mass_element(self,
+                                                                          create_default_2d_model: Model,
+                                                                          create_default_outputs: List[Output],
+                                                                          create_default_solver_settings: Problem):
+        """
+        Test correct writing of the project parameters for the default output, model and settings with a spring damper
+        and a nodal concentrated element.
+
+        Args:
+            - create_default_2d_model (:class:`stem.model.Model`): the default 2D model of a square \
+                soil layer and a line load.
+            - create_default_outputs (List[:class:`stem.output.Output`]): list of default output processes.
+            - create_default_solver_settings (:class:`stem.solver.Problem`): the Problem object containing the \
+                solver settings.
+        """
+        model = create_default_2d_model
+        kratos_io = KratosIO(ndim=model.ndim)
+        model.project_parameters = create_default_solver_settings
+        model.output_settings = create_default_outputs
+
+        # add elastic spring damper element
+        spring_damper = ElasticSpringDamper(
+            NODAL_DISPLACEMENT_STIFFNESS=[1, 1, 1],
+            NODAL_ROTATIONAL_STIFFNESS=[1, 1, 2],
+            NODAL_DAMPING_COEFFICIENT=[1, 1, 3],
```

Comment:
are you sure it is damping coefficient? units [N s/m]

Reply:
Yes, it is wrongly name in Kratos, the damping matrix in kratos is build as follows:
`C = [[c, -c],[-c, c]`